Makefile

```diff
@@ -0,0 +1,33 @@
+LDFLAGS = -ldflags "-X main.gitSHA=$(shell git rev-parse HEAD)"
```

Comment:
This should likely be removed (and subsequent referencing lines) unless you're going to set the version (git sha) in the binary itself.